doc/lsst.ap.pipe/pipeline-tutorial.rst

```diff
@@ -64,7 +64,7 @@ To process your ingested data, run
 
 .. prompt:: bash
    
-   ap_pipe.py repo --calib repo/calibs --rerun processed --id visit=123456 ccdnum=42 filter=g --template templates
+   ap_pipe.py repo --calib repo/calibs --rerun processed -c associator.level1_db.db_name=association.db --id visit=123456 ccdnum=42 filter=g --template templates
 
```

Comment:
I believe @morriscb said over on Jira that the config field should be `db_url`, not `db_name`. Unless I'm misinterpreting, please update this accordingly.

Reply:
No, the config field is `db_name`. It we for some reason had to configure `l1dbproto` then that would be relevant, but AFAIK we don't. (I can't test it because `AssociationDBSqliteTask` and `AssociationL1DBProtoTask` have incompatible APIs, and `ap_pipe` assumes the former.)